config/samples/config_v1beta1_certstore_inline.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: config.ratify.deislabs.io/v1alpha1
+apiVersion: config.ratify.deislabs.io/v1beta1
```

Comment:
leave some of the tests at v1alpha1 to make sure conversion works as expected. doesn't have to be here, just an example

Reply:
good call, I added a test which applies v1alpha1 resource.